src/main/java/pl/pandziak/ticketbookingapp/domain/Room.java

```diff
@@ -0,0 +1,25 @@
+package pl.pandziak.ticketbookingapp.domain;
+
+public enum Room {
+
+    ROOM_A(7, 10),
```

Comment:
jak będziemy mieli wielkiego multiplexa to trzymanie wielkości sal jako enum to może nie być dobry pomysł...

Reply:
Tak. Wtedy pewnie by trzeba ten enum zamienić na klasę perzystowalną. W pierwszej wersji aplikacji enum wydawał mi się wystarczający.